### PR TITLE
feat/Fund Transfer

### PR DESCRIPTION
**Pull request by shadushazzz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,13 +34,17 @@
   "author": "fe-app",
   "license": "MIT",
   "dependencies": {
-    "@lion/button": "^0.8.11",
+    "@babel/eslint-plugin": "^7.11.0",
```
Comment (Rajkumar-MP) [inline package.json]: why this plugin is added ?

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]

File: src/Components/fe-fund-transfer/FeFundTransfer.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+  static get styles() {
+    return css`
+      ${defaultStyles}
+    `;
+  }
+
+  firstUpdated() {
+    super.firstUpdated();
+    localize.locale = 'en-GB';
```
Comment (Rajkumar-MP) [inline src/Components/fe-fund-transfer/FeFundTransfer.js]: no need locale

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Nayana-2607) [reply package.json]: Will remove the plugin and update

File: src/Components/fe-fund-transfer/FeFundTransfer.js (hunk 2 above)
Comment (Nayana-2607) [reply src/Components/fe-fund-transfer/FeFundTransfer.js]: locale removed

File: package.json [hunk 3]
```diff
@@ -34,13 +34,17 @@
...
     "@lion/form": "^0.7.18",
+    "@lion/form-core": "^0.8.4",
     "@lion/input": "^0.11.1",
     "@lion/localize": "^0.16.0",
+    "@lion/select": "^0.9.4",
     "@vaadin/router": "^1.7.4",
     "lit-element": "^2.0.1",
-    "lit-html": "^1.0.0"
+    "lit-html": "^1.0.0",
+    "node-modules": "^1.0.1"
```
Comment (Rajkumar-MP) [inline package.json]: this one not needed, remove it, i already told to @alekhya66 why we dont need this one

File: src/Components/fe-fund-transfer/FeFundTransfer.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    }
+
+    this.dispatchEvent(
+      new CustomEvent('fund-validation', { detail: serializedValue })
+    );
+  }
+
+  render() {
+    return html`
+      <h1>Fund Transfer</h1>
```
Comment (Rajkumar-MP) [inline src/Components/fe-fund-transfer/FeFundTransfer.js]: Translation missing

File: src/Components/fe-fund-transfer/FeFundTransfer.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+            id="toaccount"
+            .validators="${[
+              new Required(null, {
+                getMessage: () =>
+                  localize.msg('fe-fund-transfer:accounttypeerror'),
+              }),
+            ]}"
+          >
+            <select slot="input">
+              <option selected hidden value>Please select</option>
```
Comment (Rajkumar-MP) [inline src/Components/fe-fund-transfer/FeFundTransfer.js:99]: Translation missing

File: src/FeApp.style.js [hunk 6]
```diff
@@ -69,7 +69,7 @@ export default css`
     padding: 8px;
     margin-top: 8px;
   }
-  .form--input--field input:focus {
+  .form--input--field :focus {
```
Comment (Rajkumar-MP) [inline src/FeApp.style.js]: why input is removed ?, if we do for the class directly, it will focus all ing-input fields itself

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]
Fix it or give your update for the comments

File: src/FeApp.style.js (hunk 6 above)
Comment (shadushazzz) [reply src/FeApp.style.js]: Because focus was not getting applied to select field

File: src/Components/fe-fund-transfer/FeFundTransfer.js (hunk 5 above)
Comment (shadushazzz) [reply src/Components/fe-fund-transfer/FeFundTransfer.js:99]: OK il add

File: src/Components/fe-fund-transfer/FeFundTransfer.js (hunk 4 above)
Comment (shadushazzz) [reply src/Components/fe-fund-transfer/FeFundTransfer.js]: will add and update

File: package.json (hunk 3 above)
Comment (shadushazzz) [reply package.json]: will remove it

File: src/FeApp.style.js (hunk 6 above)
Comment (shadushazzz) [reply src/FeApp.style.js]: however focus will be same to all input fields na so i thought to remove and use it for both select and input

**Review by alekhya66 — DISMISSED** [review]

**Review by Rajkumar-MP — DISMISSED** [review]

**Review by Nayana-2607 — DISMISSED** [review]

**Review by alekhya66 — DISMISSED** [review]

**Review by Nayana-2607 — APPROVED** [review]

**Review by Rajkumar-MP — APPROVED** [review]

**Review by alekhya66 — APPROVED** [review]

**Pull request merged**